src/emu/debug/express.h

```diff
@@ -179,10 +179,15 @@ class symbol_table
 	void configure_memory(void *param, valid_func valid, read_func read, write_func write);
 
 	// symbol access
-	void add(const char *name, read_write rw, u64 *ptr = nullptr);
-	void add(const char *name, u64 constvalue);
-	void add(const char *name, void *ref, getter_func getter, setter_func setter = nullptr, const std::string &format_string = "");
-	void add(const char *name, void *ref, int minparams, int maxparams, execute_func execute);
+	void add(std::string &&name, read_write rw, u64 *ptr = nullptr);
+	void add(std::string &&name, u64 constvalue);
+#ifdef _MSC_VER
```

Comment:
What do you mean by "seems not to like" in this case?  It clearly works with rvalue references in many other cases.  Why does it object to this one?

Reply:
I actually don't understand the nature of the bug.  The existence of the overload that takes an std::function as an rvalue reference seems to trigger compilation errors.  I've tried to reproduce the problem in a simpler context, but my attempts have failed.